lerobot/scripts/eval.py

```diff
@@ -155,6 +156,9 @@ def rollout(
             key: observation[key].to(device, non_blocking=device.type == "cuda") for key in observation
         }
 
+        # Infer "task" from envs. Works with AsyncVectorEnv.
+        observation = infer_envs_task(env, observation)
```

Comment:
What about SyncVectorEnv?

```suggestion
        # Infer "task" from attributes of environments. Works with AsyncVectorEnv.
        observation = addd_envs_task(env, observation)
```

Reply:
done

Reply:
typo. relaunched the test before merging?

Reply:
What about SyncVectorEnv? Does it work?

PS: Thanks for commenting, but please do not resolve the conversations ^^ That's a way for me to easily review the changes ;)